parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetWriter.java

```diff
@@ -142,6 +142,82 @@ public void test() throws Exception {
     }
   }
 
+  @Test
+  public void testQuotedColumns() throws Exception {
+    Configuration conf = new Configuration();
+    Path root = new Path("target/tests/TestParquetWriter/");
+    enforceEmptyDir(conf, root);
+    MessageType schema = parseMessageType(
+        "message test { "
+        + "required binary `binary field`; "
```

Comment:
and also tests that checks some names with special characters that worked before don't get quoted.
For backward compatibility as older versions of Parquet won't understand quotes.